webapp/src/main/java/org/opentestsystem/rdw/admin/model/School.java

```diff
@@ -0,0 +1,23 @@
+package org.opentestsystem.rdw.admin.model;
+
+import org.opentestsystem.rdw.admin.common.model.Organization;
+
+/**
+ * This model represents a school, containing the id, name, and natural id.
+ */
+public class School extends Organization {
+
+    private String naturalId;
```

Comment:
I'm pretty darn sure that every Organization has a naturalId ... that should be put there.